### DISCUSSION TITLE
create groups of people

### DISCUSSION THREAD

**Discussion by gianlucaf81** [immich-app/immich#14115] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Could you have it in Explore -> People, "People Groups"?
I assume with tags or whatever you think is best.

Example
**My family**: Me, Dad, Mom, Brother, Sister.
**Friends**: John, Kate, Robbie C. etc...
**Classmates**: Me, Maria, Tommy, Robbie C.
**Work colleagues**: Allison, Justine, Mark, Alfonso

It might also be interesting to introduce a filter in this mode that finds photos of groups with all or some of the people in the group.

_I would like to take this opportunity to congratulate you on this fantastic project. Facial recognition is something incredible._

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#6079
